newdle/client/src/components/CreateNewdle.module.scss

```diff
@@ -1,5 +1,88 @@
+@import '../styles/palette.scss';
+
 .button-row {
   display: flex;
   justify-content: flex-end;
   width: 100%;
 }
+
+.title-input {
+  width: 100%;
+  border-bottom: 1px solid $purple !important;
+  padding-bottom: 10px !important;
+  input {
```

Comment:
Maybe separate the `input` rule with the attribute in the previous line with an empty line?